<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="TypeOfRefinement" >
   <property name="geometry" >
    <rect>
     <x>128</x>
     <y>50</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>Lines</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Tube Contour</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>96</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Type Of Refinement</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Dimensions" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>80</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>2</number>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
   <property name="value" >
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Dimensions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>   Maximum Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>   Minimum Level</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MakeBounds" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Make Bounds</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="OriginCx" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>192</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="OriginCy" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SizeCx" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>192</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SizeCy" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="OriginCz" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>230</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SizeCz" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OriginCx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OriginCy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>OriginCz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SizeCx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SizeCy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>SizeCz</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NumberOfSides" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>262</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>260</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number Of sides</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Capping" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>290</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Capping</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Radius" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>320</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>320</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MaximunLevel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>110</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>2</number>
   </property>
   <property name="maximum" >
    <number>11</number>
   </property>
   <property name="value" >
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MinimumLevel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>130</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="maximum" >
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Type Of Interpolation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeOfInterpolation" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>20</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Poly Linear</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>350</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Threshold Distance</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Distance" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>350</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>16777215</number>
   </property>
   <property name="value" >
    <number>8</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
